recipes/greenlet/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "greenlet" %}
+{% set version = "0.4.12" %}
+{% set sha256 = "e4c99c6010a5d153d481fdaf63b8a0782825c0721506d880403a3b9b82ae347e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+
+test:
+  imports:
+    - greenlet
+
+about:
+  home: https://github.com/python-greenlet/greenlet
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Lightweight in-process concurrent programming'
+
+  description: |
+    The greenlet package is a spin-off of Stackless, a version of CPython that supports micro-threads called "tasklets". Tasklets run pseudo-concurrently (typically in a single or a few OS-level threads) and are synchronized with data exchanges on "channels".
+
+    A "greenlet", on the other hand, is a still more primitive notion of micro-thread with no implicit scheduling; coroutines, in other words. This is useful when you want to control exactly when your code runs. You can build custom scheduled micro-threads on top of greenlet; however, it seems that greenlets are useful on their own as a way to make advanced control flow structures. For example, we can recreate generators; the difference with Python's own generators is that our generators can call nested functions and the nested functions can yield values too. Additionally, you don't need a "yield" keyword. See the example in tests/test_generator.py.
```

Comment:
Might wrap this text a bit. It gets a bit long reading horizontally. We don't have a hard line limit, but feel free to use your best judgement. Let's say not having to scroll horizontally would be a good goal.